hack/tool/backup_restore/cmd/restore.go

```diff
@@ -0,0 +1,170 @@
+/*
+Copyright 2022 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/oam-dev/terraform-controller/api/v1beta2"
+	"github.com/oam-dev/terraform-controller/controllers/configuration/backend"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+	v1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	stateJSONPath     string
+	configurationPath string
+	k8sClient         client.Client
+)
+
+// newRestoreCmd represents the restore command
+func newRestoreCmd(kubeFlags *genericclioptions.ConfigFlags) *cobra.Command {
+	restoreCmd := &cobra.Command{
+		Use: "restore",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var err error
+			k8sClient, err = buildClientSet(kubeFlags)
+			if err != nil {
+				return err
+			}
+			pwd, err := os.Getwd()
+			if err != nil {
+				return err
+			}
+			stateJSONPath = filepath.Join(pwd, stateJSONPath)
+			configurationPath = filepath.Join(pwd, configurationPath)
+			return restore(context.Background())
+		},
+	}
+	restoreCmd.Flags().StringVar(&stateJSONPath, "state", "state.back.json", "the path of the backed up Terraform state file")
+	restoreCmd.Flags().StringVar(&configurationPath, "configuration", "configuration.back.yaml", "the path of the backed up configuration objcet yaml file")
```

Comment:
```suggestion
	restoreCmd.Flags().StringVar(&configurationPath, "configuration", "configuration.yaml", "the path of the backed up configuration objcet yaml file")
```
Normally the configuration file is there, no need to have a backup.